### ISSUE TITLE
glusterfs volume name

### ISSUE THREAD

**Issue by wiryonolau** [trajano/docker-volume-plugins#24]
Hi, does glusterfs volume name require to be "gfs" ?
Can I use different name, how do I specifiy it in the options ?

**Comment by trajano** [comment]
I don't think so, there's nothing that says `gfs` in https://github.com/trajano/docker-volume-plugins/tree/master/glusterfs-volume-plugin
